app/views/jobs/_form.html.erb

```diff
@@ -1,15 +1,31 @@
+<% if @error %>
+    <h2><%= pluralize(@error.count, "error") %>
+        prohibited this record from being saved:</h2>
+    <ul>
+      <% @error.full_messages.each do |message| %>
+      <li><%= message %></li>
+      <% end %>
+    </ul>
+    <% end %>
+
 <%= form_for [@company, @job] do |f| %>
   <%= f.label :title %>
-  <%= f.text_field :title %>
+  <%= f.text_field :title ,class: "form-control"%>
 
   <%= f.label :description %>
-  <%= f.text_field :description %>
+  <%= f.text_field :description, class: "form-control" %>
 
   <%= f.label :level_of_interest %>
-  <%= f.number_field :level_of_interest %>
+  <%= f.number_field :level_of_interest, class: "form-control" %>
 
   <%= f.label :city %>
-  <%= f.text_field :city %>
-
+  <%= f.text_field :city, class: "form-control"%>
+  <% if Category.count > 0 %>
```

Comment:
this is a lot of logic in your view. we're both counting all the categories and also seeing if the count is greater than zero. i suggest setting an instance variable in your controller to all the categories... then, we can say `if @categories.any?` here and on line 25, we can access `@categories` rather than again querying the database